modules/web/src/app/shared/components/terminal/template.html

```diff
@@ -38,10 +38,13 @@
                          (openInNewTab)="openInSeparateView()"
                          (close)="onClose()"></km-terminal-toolbar>
 
-    <km-terminal-status-bar *ngIf="isConnectionLost"
-                            [showCloseButton]="showCloseButtonOnToolbar"
+    <km-terminal-status-bar *ngIf="isConnectionLost || isSessionExpiring"
+                            [isConnectionLost]="isConnectionLost"
+                            [isSessionExpiring]="isSessionExpiring"
                             (reconnect)="onReconnect()"
-                            (close)="onClose()"></km-terminal-status-bar>
+                            (extendSession)="onExtendSession()"
+                            (close)="onClose()">
```

Comment:
I see that `close` Output is removed from `km-terminal-status-bar` component so we should remove this handle as well.